### ISSUE TITLE
Buff list and subtitles covered issue

### ISSUE THREAD

**Issue by IsaacRice** [Kamikaze94/WolfHUD#39]
Hi, I am the guy asking about the misplaced subtitles issue from the lastbullet.

I tried the dev build and found out that subtitles still covered with buff list.

Currently it works perfect with the buff icon.
But it still covers the buff text above the buff icon like +Dmg or +M.Dmg.

Since they are different color, it's still hard to read the subtitles.

Please fix it, thanks.

BTW, I don't really understand some option in settings.
I'll list below.

Show Callsign - not figure out what exactly Callsign means.
Light Multiplier - not understand and it seems do nothing when changing the parameter
Glow Divider - not understand and it seems do nothing when changing the parameter

**Comment by Kamikaze94** [comment]
Hey there.
Yes, I remember your name, hi :P
Well it didn't for me. Which Buff List scale are you using, and did you change it on the same map, you noticed the overlap?
Maybe you wanna post a screenshot of the issue as well (just drop the file into the comment box, to attach it :)

For your questions:
The callsign is the colored sicle in front of that players name. 
(In center of the health, if Name, rank, character, etc is idisabled, can make down counter hard to read...)
Its not to easy to explain, but the laser basically consists of the actual laser beam, the dot where it hits a surface and the glow around that dot.
The light multiplier makes the dot brighter by default, so it gets better visible.
The glow divider reduces the color and brightness of the glow around the dot.
(At least thats what I think how it works...  :P)

**Comment by IsaacRice** [comment]
Here is the [Screenshot](http://imgur.com/a/AYWUU).

My Buff List scale is set to 0.9 and I've already tried to change the scale.
It still covered with the text of Berserker.

Thank for your explanation.
I know the reason why I can't figure out what Callsign do.
I didn't test it with a teammate. lol

About the light multiplier and the glow divider, are these two fine-tuning?
I really can't find the difference between I change the parameter.
Or should it restart the game? Currently I only restart the mission to applied the change.

**Comment by Kamikaze94** [comment]
Okay, the Subtitle still overlapping was caused by some outdated code in HUDList and some missing code in CustomHUD.
Got that fixed now.
Thanks for reporting that bug.

And yes, the options for the multiplier and divider for lasers came after I added some code for laser color instant update, so it will probably need a (mission-) restart to apply.
The difference is pretty minor, so it might be hard to spot.
Oh, and only player and senty lasers have a dot at the end, as far as I remember.
